Update working-with-sql.md (#24760)
dotnet/AspNetCore.Docs: aspnetcore/tutorials/first-mvc-app/working-with-sql.md

Replace the contents of *Program.cs* with the following code. The new code is highlighted.

[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie60/Program.cs?name=SQLServerSeedData&highlight=4,16-21)]

Delete all the records in the database. You can do this with the delete links in the browser or from SSOX.

Test the app. Force the app to initialize, calling the code in the *Program.cs* file, so the seed method runs. To force initialization, close the command prompt window that Visual Studio opened, and restart by pressing Ctrl+F5.

# [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)

> Update *Program.cs* with the following highlighted code.

Reviewer: ```suggestion
Update *Program.cs* with the following highlighted code:
```

> Replace the contents of *Program.cs* with the following code. The new code is highlighted.

Reviewer: ```suggestion
Update *Program.cs* with the following highlighted code.
```
